plugins/action/ci_script.py

```diff
@@ -122,6 +122,13 @@
 
 
 class ActionModule(ActionBase):
+    __OUTPUT_DIR_ARG = "output_dir"
+    __SCRIPT_ARG = "script"
+    __DRY_RUN_ARG = "dry_run"
+    __EXTRA_ARGS_ARG = "extra_args"
+    __CHDIR_ARG = "extra_args"
```

Comment:
I think it should be:
```suggestion
    __CHDIR_ARG = "chdir"
```